recipes/mpc.mdwn

```diff
@@ -52,10 +53,15 @@ updates when the current MPD state changes.
         end
         mpd_widget.text = text
     end
+    local connection
     local function error_handler(err)
         mpd_widget:set_text("Error: " .. tostring(err))
+        -- Try a reconnect soon-ish
+        timer.start_new(60, function()
```

Comment:
Depends on how often you want to see the error message. ;-)
Best would be something like (almost-)exponential backoff, but that would be complicated. It's just an arbitrary number. Perhaps "1 minute" feels a bit less arbitrary than 10 seconds. I don't know and don't care much. If you want, I'll turn it into 10 seconds.